libyul/backends/evm/EVMDialect.cpp

```diff
@@ -378,6 +378,7 @@ EVMDialectTyped::EVMDialectTyped(langutil::EVMVersion _evmVersion, bool _objectA
 		BuiltinContext&,
 		std::function<void(Expression const&)> _visitExpression
 	) {
+		// TODO this should use a Panic.
```

Comment:
In this PR?

Reply:
No, this dialect is not really used yet.

Reply:
Ok